src/mapml-viewer.js

```diff
@@ -31,6 +31,18 @@ export class MapViewer extends HTMLElement {
     if (this.controlslist.includes(lowerVal) || !options.includes(lowerVal))return;
     this.setAttribute("controlslist", this.controlslist+` ${lowerVal}`);
   }
+  get width() {
+    return this.hasAttribute('width') ? this.getAttribute("width") : window.getComputedStyle(this).width;
```

Comment:
The `window.getComputedStyle(this).width` returns width in the following format '100px' while `this.getAttribute("width")` returns '100', maybe you can add something like `.replace('px','')` to keep things consistent. Same thing for `get height()`.